apps/coscrad-frontend/src/components/Home/Home.tsx

```diff
@@ -1,13 +1,20 @@
+import { useDispatch } from 'react-redux';
 import { Link } from 'react-router-dom';
+import { fetchResourceInfos } from '../../store/slices/ResourcesSlice';
 import AuthenticationButton from '../AuthenticationButton/AuthenticationButton';
 import './Home.module.scss';
 
 export function Home() {
+    const dispatch = useDispatch();
+
     return (
         <div>
             <h1>Welcome to coscrad</h1>
-            <Link to="/About">About</Link> | <Link to="/AllResources">Browse Resources</Link> |
-            <Link to="/MembersOnly">Members Only</Link>
+            <Link to="/About">About</Link> |{' '}
+            <Link to="/AllResources" onClick={() => dispatch<any>(fetchResourceInfos())}>
```

Comment:
Shouldn't we do the fetch when the all resources page loads? Or better yet, check if Redux has the data then fetch if not.

Reply:
Fixed.